src/pages/ProductList/ProductList.js

```diff
@@ -1,67 +1,28 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import "./ProductList.scss";
 
 const ProductList = () => {
+  const [productList, setProductList] = useState();
+
+  useEffect(() => {
+    fetch("data/productlist.json", { method: "GET" })
+      .then((response) => response.json())
+      .then((data) => setProductList(data.list));
```

Comment:
```suggestion
    fetch("data/productlist.json")
      .then((response) => response.json())
      .then((data) => setProductList(data.list));
```
method가 GET 요청이고, 다른 조건이 없다면 해당 부분은 생략하셔도 됩니다!